sheep/cluster/local.c

```diff
@@ -307,15 +307,17 @@ static int add_event(enum local_event_type type, struct local_node *lnode,
 		xlremove(lnode, ev.lnodes, &ev.nr_lnodes, lnode_cmp);
 		break;
 	case EVENT_GATEWAY:
-		if (n = xlfind(lnode, ev.lnodes, ev.nr_lnodes, lnode_cmp))
+		if ((n = xlfind(lnode, ev.lnodes, ev.nr_lnodes, lnode_cmp))) {
```

Comment:
How about
(n = xlfind(lnode, ev.lnodes, ev.nr_lnodes, lnode_cmp)) != NULL
?
Just for making the intention clearer.